ext4/tar2ext4/tar2ext4.go

```diff
@@ -18,6 +18,12 @@ import (
 	"github.com/Microsoft/hcsshim/ext4/internal/format"
 )
 
+const (
+	// default maximum size in GB of any layer
+	// TODO: upstream with maxLcowVhdSizeGB in containerd 
```

Comment:
we should remove this comment as well and put a generic message explaining this constant.

Reply:
OK, what explanation should I put? 
```
This constant is for internal use. It is not a hard limit.
```